schemas/account.js

```diff
@@ -23,5 +23,21 @@ const getAccountsQuery = {
   }
 };
 
+const setStatus = {
+  title: 'bedrock-accounts-http patch set status',
+  required: ['status'],
+  type: 'object',
+  additionalProperties: false,
+  properties: {
+    status: {
+      type: 'string',
+      pattern: '^(active|deleted)$',
```

Comment:
`enum` is what you want here instead of pattern, no need for minLength/maxLength then either
```suggestion
      enum: ['active', 'deleted']
```

Reply:
I knew it had to be there doing it now.